sdk/src/azure/iot/az_iot_hub_client_properties.c

```diff
@@ -17,7 +17,7 @@ static const az_span properties_ack_description_name = AZ_SPAN_LITERAL_FROM_STR(
 static const az_span component_properties_label_name = AZ_SPAN_LITERAL_FROM_STR("__t");
 static const az_span component_properties_label_value = AZ_SPAN_LITERAL_FROM_STR("c");
 
-AZ_NODISCARD az_result az_iot_hub_client_properties_update_get_publish_topic(
+AZ_NODISCARD az_result az_iot_hub_client_properties_get_reported_publish_topic(
```

Comment:
We are now somewhat inconsistent with others like `az_iot_hub_client_properties_document_get_publish_topic`

Can we change those too or have they all GA'd already?

Reply:
Everything in `az_iot_hub_client_properties.c` is still in beta, so we have until GA to make changes.

In parallel we're reconsidering using the "writable" property naming in API itself.  It's great it's consistent with DTDL but further research/discussion with other languages doing this same work suggests the benefit of this isn't worth the somewhat unnatural flavor this from a device-1st perspective. 

Some folks on broader (IoT) SDK team are going to come up with some alternatives and we'll absolutely re-cover this with Azure SDK once we start getting something more concrete. 

It's not something we should hold beta on so I'm resolving for now.